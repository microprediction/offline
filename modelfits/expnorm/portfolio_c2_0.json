{"g1": 0.08420888535371229, "g2": 14.817578955713728, "logK": 0.33062213633711757, "loc": -0.023171925985431635, "logScale": 0.4434774005787747}