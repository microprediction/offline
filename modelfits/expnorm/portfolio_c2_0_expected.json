{"g1": 0.266303131430418, "g2": 0.866321827446691, "logK": 0.568478037633962, "loc": -0.11512271833774301, "logScale": -1.8131900952589477}